### PR TITLE
[For Review] Separate pom.xml dependencies

### PR DESCRIPTION
**Pull request by ikeyat**
Don't merge. Just for review.
Please review #508.

### CODE REVIEW

**Comment by kazuki43zoo** [comment]
@ikeyat @sandeepunde 

Please modify to use space 4 characters as indent ('Tab' does not use).
Currently, it is not easy to check essential changes(diff).

File: terasoluna-gfw-starters/terasoluna-gfw-starter-jodatime/pom.xml [hunk 1]
```diff
@@ -7,7 +7,7 @@
 		<groupId>org.terasoluna.gfw</groupId>
 		<artifactId>terasoluna-gfw-starters</artifactId>
 		<version>5.2.0-SNAPSHOT</version>
-		<relativePath>../terasoluna-gfw-starters/pom.xml</relativePath>
+		<relativePath>../../terasoluna-gfw-starters/pom.xml</relativePath>
```
Comment (ikeyat) [inline terasoluna-gfw-starters/terasoluna-gfw-starter-jodatime/pom.xml]: - [ ] Why don't you use ``../pom.xml``?

Comment (btundes) [inline terasoluna-gfw-starters/terasoluna-gfw-starter-jodatime/pom.xml]: @ikeyat 
By using ```<relativePath>../../terasoluna-gfw-starters/pom.xml</relativePath> ``` , I have tried to follow the approach used in the artifact like ```terasoluna-gfw-codepoints-jisx0201``` .

However, I feel that the value ``` ../pom.xml``` and ```../../terasoluna-gfw-starters/pom.xml``` used in the relativePath tag of each of the starter pom refers to the ```pom``` from ```terasoluna-gfw-starters```

**Comment by ikeyat** [comment]
These changes have been merged at #557

**Pull request closed without merging**